app/renderer/js/components/handle-external-link.ts

```diff
@@ -22,8 +20,6 @@ function handleExternalLink(this: any, event: any): void {
 	} = LinkUtil.isInternal(domainPrefix, url);
 
 	if (isWhiteListURL) {
-		event.preventDefault();
```

Comment:
Can't remember exactly why we added this but I think it has something to do with the race condition where the link was being opened in the browser + the desktop app. Please cross-check it.

Reply:
For what type of link do i need to test this?

Reply:
Try to test the following links -
* Internal url - Click on internal zulip link like send a message to yourself with something like https://chat.zulip.org/#narrow/stream/112-desktop-issues and click the same.
* Click on an attachment link.
* Some external URL.

Reply:
@vsvipul Is this being tested?